app/models/mixins/vim_connect_mixin.rb

```diff
@@ -42,6 +42,7 @@ def raw_connect(options)
     end
 
     def validate_connection
+      require 'handsoap'
```

Comment:
Don't you need to use a `begin` / `rescue` / `end` here if you're going to require and rescue inside the method?

Reply:
Apparently not :)

Reply:
@bdunne Sorry - what are you suggesting here?

That I add a `begin` block type construct?

I'm confused why this change would fix on mine. maybe it was a 2.4 thing.
(no pushback - just want to add what you think is necessary)

Reply:
This is called from `raw_connect` per your [backtrace](https://github.com/ManageIQ/manageiq/pull/16667#issuecomment-351776443), and we already require handsoap [there](https://github.com/kbrock/manageiq/blob/e4c1248452d5676ed055c83a058a41b205123d55/app/models/mixins/vim_connect_mixin.rb#L35) so a. I don't know why this failed if we already require it and b. requiring it again probably won't fix whatever is wrong